src/cfnlint/rules/resources/ResourceSchema.py

```diff
@@ -0,0 +1,32 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: MIT-0
+"""
+import os
+import re
+from glob import glob
+from jsonschema import validate, ValidationError
```

Comment:
One of the big problems we are going to have with using this package is handling `Fn::If` or other intrinsic functions. Either way we are going to have to handle it outside of the schema validation which we have logic to handle or we are going to have to rewrite these validation functions that this package provides to handle these things differently.